React/Base/RCTUIKit.h

```diff
@@ -482,6 +482,30 @@ NS_INLINE CGRect CGRectValue(NSValue *value)
 @end
 #endif // ]TODO(macOS GH#774)
 
+// RCTUIActivityIndicatorView
+
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
+#define RCTUIActivityIndicatorView UIActivityIndicatorView
+#else
+@interface RCTUIActivityIndicatorView : UIActivityIndicatorView
```

Comment:
Ah.. shouldn't this be [NSProgressIndicator](https://developer.apple.com/documentation/appkit/nsprogressindicator?language=objc) ? See the paper component for example.

Reply:
fixed in 91d707454721ba4055606864561324022940959c

Reply:
I think I prefer bring explicit anyway, so thanks!